website/docs/d/availability_set.html.markdown

```diff
@@ -0,0 +1,48 @@
+---
```

Comment:
can we add a link to this in the sidebar? that's done here https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/website/azurerm.erb#L72-L88